Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -2857,6 +2852,7 @@ recenterCurrentLocation :: HomeScreenState -> Eval Action ScreenOutput HomeScree
 recenterCurrentLocation state = continueWithCmd state [ do
     if state.props.locateOnMap || (not state.props.locateOnMap && state.props.currentStage == ConfirmingLocation) then do
       _ <- pure $ currentPosition "NO_ZOOM"
+      _ <- addMarker (getCurrentLocationMarker (getValueToLocalStore VERSION_NAME)) 9.9 9.9 160 (0.5) (0.9)
```

Comment:
@yashikakaushik calling this function here will cause double marker issue. If the issue is with marker not being added in homeScreen then try calling it in MapReadyAction.